lib/install/javascript/initializers/index.js

```diff
@@ -0,0 +1,2 @@
+const importAll = (r) => r.keys().forEach(r)
+importAll(require.context('.', true, /\.js$/));
```

Comment:
Will this end up importing itself (`./index.js`)?

Reply:
Yes, though my understanding is that is [supported in ES6-style module imports](http://exploringjs.com/es6/ch_modules.html#sec_cyclic-dependencies).  

I created a [gist](https://gist.github.com/rossta/5a9edcd7ba37416f8c6f7ed383200b0d) that includes an excerpt of the Webpack output to show that the result would be similar to placing `import './index'` and `import './hello'` in `initializers/index.js`, which does complie and execute both modules.

If preferred, I can move the `require.context` code to a separate file.